Tasks/google-play-release/GooglePlay.js

```diff
@@ -27,6 +27,17 @@ if (authType === "JsonFile") {
 }
 
 var apkFile = resolveGlobPath(tl.getPathInput("apkFile", true));
+var apkFileList = [apkFile];
+var additionaApks = tl.getDelimitedInput('additionalApks', '\n');
+if (additionaApks.length > 0) {
+    for (var i in additionaApks) {
+        apkFileList.push(resolveGlobPath(additionaApks[i]));
```

Comment:
After speaking with Ruben, it seems like the current behavior is by design, and we can always change it later if needed.